e2e/watch-runner.ts

```diff
@@ -42,7 +41,7 @@ export default class WatchRunner {
     });
   }
   async waitForWatchToRebuildComponent() {
-    return this.waitForWatchToPrintMsg(WATCHER_COMPLETED_MSG);
+    return this.waitForWatchToPrintMsg('');
```

Comment:
this will probably break the tests

Reply:
What is the right way to do it?

Reply:
as above, after there is a file change and the watch is triggered, on finish it writes something to the screen (probably something like `finished` or `watching for component changes` or whatever)
you should put this string here.

Reply:
If I've got you correctly this is not possible, because of the timestamps.

Reply:
Fixed.